packages/venia-concept/src/actions/cart/asyncActions.js

```diff
@@ -21,17 +21,16 @@ export const createGuestCart = () =>
         // in case the user has already completed an order this session
         dispatch(checkoutActions.reset());
 
-        const guestCartId = await retrieveGuestCartId();
+        // Request a new guest cart.
+        dispatch(actions.getGuestCart.request());
 
         // if a guest cart exists in storage, act like we just received it
+        const guestCartId = await retrieveGuestCartId();
```

Comment:
This isn't anything special, I just rearranged the dispatches for consistency with other actions.

Previously we'd get an `actions.getGuestCart.receive` without a preceeding `actions.getGuestCart.request` when we already had a `guestCartId`.